cmd/contact-exporter/main.go

```diff
@@ -23,11 +21,17 @@ type contactExporter struct {
 	clk   clock.Clock
 }
 
-func (c contactExporter) findContacts() ([]string, error) {
-	var contactsJSON [][]byte
+type contactJSON struct {
+	ID      int64  `json:"id"`
+	Contact []byte `json:"-"`
+}
+
+// Find all registration contacts with unexpired certificates.
+func (c contactExporter) findContacts() ([]contactJSON, error) {
+	var contactsList []contactJSON
 	_, err := c.dbMap.Select(
-		&contactsJSON,
-		`SELECT contact AS active_contact
+		&contactsList,
+		`SELECT id
```

Comment:
Nope - it just exports the reg IDs now, no need to look at the `contact` field.

The `notify-mailer` cmd later selects using these IDs to retrieve the `contact` JSON.